data/faq/application_template.md

```diff
@@ -0,0 +1,86 @@
+coala 2017 Application Template
```

Comment:
they can't copy this template since it's going to be rendered so you'll want to make a code block and optimize for plain text readability!